docs/source/en/preprocessing.mdx

```diff
@@ -387,32 +387,32 @@ For computer vision tasks, it is common to add some type of data augmentation to
 4. Now when you access the image, you'll notice the image processor has added `pixel_values`. You can pass your processed dataset to the model now!
 
 ```py
->>> dataset[0]["image"]
-{'image': <PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=384x512 at 0x7F1A7B0630D0>,
+>>> dataset[0]
+{'image': <PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=384x512 at 0x7F0B133FDA90>,
  'label': 6,
- 'pixel_values': tensor([[[ 0.0353,  0.0745,  0.1216,  ..., -0.9922, -0.9922, -0.9922],
-          [-0.0196,  0.0667,  0.1294,  ..., -0.9765, -0.9843, -0.9922],
-          [ 0.0196,  0.0824,  0.1137,  ..., -0.9765, -0.9686, -0.8667],
+ 'pixel_values': tensor([[[-0.0824, -0.0902, -0.1216,  ..., -0.1843, -0.2549, -0.2392],
```

Comment:
Showing this is taking a lot of space for nothing. Maybe just show `dataset[0].keys()`?